net-irc/inspircd/inspircd-3.4.0.ebuild

```diff
@@ -0,0 +1,123 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit toolchain-funcs
+
+DESCRIPTION="Inspire IRCd - The Stable, High-Performance Modular IRCd"
+HOMEPAGE="https://inspircd.github.com/"
+SRC_URI="https://github.com/inspircd/inspircd/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~ppc64 ~x86"
+IUSE="debug gnutls ldap maxminddb mbedtls mysql pcre postgres re2 regex-posix regex-stdlib sqlite ssl sslrehashsignal tre"
+
+RDEPEND="
+	acct-group/inspircd
+	acct-user/inspircd
+	dev-lang/perl
+	gnutls? ( net-libs/gnutls:= dev-libs/libgcrypt:0 )
+	ldap? ( net-nds/openldap )
+	maxminddb? ( dev-libs/libmaxminddb )
+	mbedtls? ( net-libs/mbedtls:= )
+	mysql? ( dev-db/mysql-connector-c:= )
+	pcre? ( dev-libs/libpcre )
+	postgres? ( dev-db/postgresql:= )
+	re2? ( dev-libs/re2:= )
+	sqlite? ( >=dev-db/sqlite-3.0 )
+	ssl? ( dev-libs/openssl:= )
+	tre? ( dev-libs/tre )"
+DEPEND="${RDEPEND}"
+
+DOCS=( docs/. )
```

Comment:
Hmm, does this really work? Legit question, I have a feeling I tried this once and it didn't work.

Reply:
Looks like it:

```
test /usr/portage/net-irc/inspircd # ls /usr/share/doc/inspircd-3.4.0/ 
Doxyfile.bz2  LICENSE.txt.bz2  conf  sql
test /usr/portage/net-irc/inspircd # ls /home/frostsnow/inspircd/docs/ 
Doxyfile  LICENSE.txt  conf  sql
```

Reply:
(I think I got it - FYI for anyone interested: I tried with `*`, not `.`)